scripts/skills/perks/perk_rf_supporter.nut

```diff
@@ -5,7 +5,8 @@ this.perk_rf_supporter <- this.inherit("scripts/skills/skill", {
 		ActionPointsRecovered = 2,
 
 		// Private
-		IsSpent = false
+		IsSpent = false,
+		RecoveredActionPoints = 0
```

Comment:
I suggest turning this into `WillRecoverActionPoints = false`.
The logic further down will be similar that you check the adjacency condition before you execute the skill. But you do not calculate the recovered action points by that time because it won't be accurate/as expected